src/local_identifiability.jl

```diff
@@ -314,21 +325,21 @@ function assess_local_identifiability(ode::ODE{P}, funcs_to_check::Array{<: Any,
 
     @debug "Computing the result"
     base_rank = LinearAlgebra.rank(Jac)
-    result = Array{Bool,1}()
+    result = Array{Pair{Any, Bool},1}()
```

Comment:
Why do we have an array here, not a dict right away?

Reply:
Same as in the other part, I tried making code changes minimal so as not to break anything